server/bin/gold/test-26.3.txt

```diff
@@ -0,0 +1,112 @@
++++ Running test_logger_type.py
+BadConfig exception was raised
+--- Finished test_logger_type.py (status=0)
```

Comment:
We probably should exit with status 1 here.

Reply:
Yaa